lib/client.js

```diff
@@ -1575,6 +1575,15 @@ async function writeIterable ({ client, request, socket, contentLength, header,
     cb()
   }
 
+  function onError (err) {
+    // TODO (fix): Avoid using err.message for logic.
+    if (err && (err.code !== 'UND_ERR_INFO' || err.message !== 'reset')) {
+      util.destroy(body, err)
```

Comment:
are you sure about this? body is an `iterable` and not a stream, so it doesn't need to get destroyed

Reply:
Good point.